packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -229,6 +297,10 @@ export async function generatePersistedQueryManifest(
     }
   }
 
+  if (configFile) {
+    maybeReportErrorsAndExit(configFile);
+  }
```

Comment:
```suggestion
  maybeReportErrorsAndExit(configFile);
```

Reply:
Good call! Consolidated in [8f2d5dd](https://github.com/apollographql/apollo-utils/pull/309/commits/8f2d5dd5cb474dd07d6abb182d98a730489998b1)